python/MooseDocs/__init__.py

```diff
@@ -19,8 +19,8 @@
     MSG = "MooseDocs requires anytree (http://anytree.readthedocs.io/en/latest/index.html)\n"
     MSG += "version 2.4.0 or greater. If you are using the MOOSE environment package\n"
     MSG += "you can upgrade by running the following command.\n"
-    MSG += "    pip install --upgrade --user anytree"
-    print MSG
+    MSG += "    pip3 install --upgrade --user anytree"
```

Comment:
Maybe use \t, and possibly just say pip. It's still unclear what the masses will name these version stricken tools when python2 goes away.